Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -1417,7 +1454,7 @@ script:
       description: Description of the signature.
       type: string
     description: Retrieve a binary from CarbonBlack based on hash. Returned file will
-      be places in the war room.
+      be places in the war room. Deprecated - please use cb-binary-download instead.
```

Comment:
we should first mention the deprecated section, and only then the explanation itself

Reply:
done